R/gains-and-losses.R

```diff
@@ -26,6 +49,11 @@ get_realized_gains_losses <- function(account_number) {
     content(as = "text", encoding = "UTF-8") |>
     parse_json()
 
+  if (isTRUE(response_json[["pageSize"]] == 0)) {
+    message('Response had pageSize of 0, no realized/gains losses found.\nThis may indicate that no "SELL" activities have ocurred in this account.')
+    return(tibble())
+  }
+
```

Comment:
Found a bug here where accounts that don't have selling activity (for example, a retirement account where it's unlikely you're selling positions) throw an error about `.data$symbol` not being found. 

Implemented an early return here for those cases and tried to provide a helpful error message. The function returns an empty tibble in these cases to keep consistency.